caluma/tests/__snapshots__/test_schema.ambr

```diff
@@ -4361,6 +4392,12 @@
     id: ID!
   }
   
+  """An enumeration."""
```

Comment:
A better description would be helpful

Reply:
This is autogenerated by Graphene if no docstring is provided